files/scripts/chassis_module_config.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
```

Comment:
why do we another service to handle CLI command?
We can call the platform script from the CLI handler?

Reply:
In this solution, chassis_module_config.py needs to subscribe the "CHASSIS_MODULE" tables to shutdown/startup a SFM and related swss/syncd services other than directly calls the platform script from the CLI handler. This solution has some considerations, please refer to https://github.com/sonic-net/sonic-buildimage/pull/18578#issuecomment-2045550189. Thanks.

Reply:
> why do we another service to handle CLI command? We can call the platform script from the CLI handler?

@arlakshm  For the case --  If user shutdowns the SFM and save the configuration file.  And reboot the whole chassis, do we want to keep the SFM down after the system is up?  If we don't need to handle this case, we can directly call the platform script from the CLI handler.